neutron.cpp

```diff
@@ -0,0 +1,172 @@
+/* 
+ @file      neutron.cpp
+ @brief     contains functions for the Neutron class
+ @author    Luke Eure
+ @date      January 8 2016
+*/
+
+#include "neutron.h"
+
+/*
+ @brief constructor for Neutron class
+ @param position a vector containing the starting position of the neutron
+ @param theta the polar angle of the neutron's initial direction
+ @param phi the azimuthal angle of the neutron's initial direction
+*/
+Neutron::Neutron(std::vector <double> &position, double theta, double phi) {
+    _xyz = position;
+    _neutron_alive = true;
+    _neutron_direction.resize(3);
+    setDirection(theta, phi);
+}
+
+/*
+ @brief deconstructor for Neutron class
+*/
+Neutron::~Neutron() {}
+
+/*
+ @brief tells if the neutron is alive
+ @return a bool (true=neutron alive. false=neutron dead)
+*/
+bool Neutron::alive(){
+    return _neutron_alive;
+}
+
+/*
+ @brief returns the energy group of the neutron
+ @return an int: the energy group of the neutron
+*/
+int Neutron::getGroup() {
+    return _neutron_group;
+}
+
+/*
+ @brief moves the neutron a given distance
+ @param distance the distance the neutron should be moved
+*/
+void Neutron::move(double distance) {
+    for (int i=0; i<3; ++i) {
+        _xyz[i] += _neutron_direction[i]*distance;
```

Comment:
I would recommend leaving a space between the multiplication since you do elsewhere